testcase.py

```diff
@@ -1190,3 +1292,24 @@ def _get_full_current_state(self):
         z.update(self.u)
 
         return z
+
+    def to_camel_case(self, snake_str):
```

Comment:
I'm sorry we have to do these conversions.

Reply:
Maybe this gets cleaned up down the line or something.  Fine for now.